internal/controller/controllers/common.go

```diff
@@ -507,3 +508,26 @@ func spokeKubeconfigSecret(ctx context.Context, log logrus.FieldLogger, c client
 
 	return secret, nil
 }
+
+func ensureVeleroExcludeBackupLabel(ctx context.Context, log logrus.FieldLogger, c client.Client, obj client.Object) error {
+	labels := obj.GetLabels()
+	if labels == nil {
+		labels = make(map[string]string)
+	}
+
+	if val, ok := labels[VeleroExcludeBackupLabel]; ok && val == "true" {
+		return nil
+	}
+
+	patch := client.MergeFrom(obj.DeepCopyObject().(client.Object))
```

Comment:
nit: can it be simplified by using Update instead (with metav1.SetMetaDataLabel). E.g. as in: https://github.com/openshift/assisted-service/blob/master/internal/controller/controllers/common.go#L85
Or do we prefer partial modification since it should be the best-practice for a label...
If so, then maybe worth just using the SetMetaDataLabel func.